package.json

```diff
@@ -72,6 +71,7 @@
     "lodash.snakecase": "^4.1.1",
     "merge-stream": "^1.0.1",
     "methmeth": "^1.1.0",
+    "pewl": "^0.1.0",
```

Comment:
where is this library?

Reply:
This is something I put together specifically for Spanner - https://github.com/callmehiphop/pewl.

Reply:
I see. Why not put it here itself?

Reply:
I wasn't able to find a pooling library that met all our requirements so I decided it might be beneficial to the Node community if I opened sourced it in case other developers were searching for similar functionality.

Reply:
Can we put it in `nodejs-spanner` for now, please?

Reply:
@lukesneeringer would giving you and @vkedia ownership of the repo suffice instead?

Reply:
I think it is more than ownership. Session pooling is a pretty core piece of cloud spanner library. It will be much more flexible if that library is specific to spanner and not open to the world to use. That will give us much more leeway in changing it/enhancing it as per our requirements.

Reply:
Any updates here?

Reply:
Between the request to move the pooling logic to this repo and a few work re-prioritizations I've been a little set back. This is my main focus now and I should have something for you to review later today.

Reply:
@vkedia I've made a sizable commit with the pooling refactor, I still need to write tests but if you'd be kind enough to review while I'm working on them, that would be awesome!